README.md

```diff
@@ -68,9 +68,9 @@ Blazor Menu control supports several built-in themes such as Tailwind CSS, Boots
 The Syncfusion's [Blazor components](https://www.syncfusion.com/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-menu-bar-github-samples) library offers over 70 UI components to work with Blazor server-side and client-side (Blazor WebAssembly) projects seamlessly. In addition to Menu Bar, we provide popular Blazor Components such as [DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid?utm_source=github&utm_medium=listing&utm_campaign=blazor-menu-bar-github-samples), [Charts](https://www.syncfusion.com/blazor-components/blazor-charts?utm_source=github&utm_medium=listing&utm_campaign=blazor-menu-bar-github-samples), [Scheduler](https://www.syncfusion.com/blazor-components/blazor-scheduler?utm_source=github&utm_medium=listing&utm_campaign=blazor-menu-bar-github-samples), [Diagram](https://www.syncfusion.com/blazor-components/blazor-diagram?utm_source=github&utm_medium=listing&utm_campaign=blazor-menu-bar-github-samples), and [Word Processor](https://www.syncfusion.com/blazor-components/blazor-word-processor?utm_source=github&utm_medium=listing&utm_campaign=blazor-menu-bar-github-samples).
```

Comment:
over 80 UI components.......
<img width="535" alt="image" src="https://github.com/SyncfusionExamples/create-blazor-menu-bar-component-in-blazor-server-app/assets/61148953/10d4c9e6-1711-447b-afe6-d46d3bf53d04">

Reply:
changed